common/membership/rpServiceResolver.go

```diff
@@ -295,3 +300,35 @@ func (r *ringpopServiceResolver) getLabelsMap() map[string]string {
 	labels[RoleKey] = r.service
 	return labels
 }
+
+func ExternalAddressResolver(listenerPeerInfo tchannel.LocalPeerInfo, broadcastIP *string) (string, error) {
```

Comment:
Let's do some renames:
1. `ExternalAddressResolver` to `BuildBroadcastHostPort`
2. `broadcastIP` to `broadcastAddress` (everywhere)